check_endpoints.py

```
@@ -0,0 +1,101 @@
+        response = requests.post(
+            url, 
+            json={"email": email, "password": password},
+            timeout=5
+        )
+        if response.status_code == 200:
+            return response.json().get("access_token")
+        else:
+            print(f"Login failed with status {response.status_code}")
+            try:
+                print(f"Error: {response.json()}")
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)